stacker/tests/fixtures/vpc-custom-log-format-info.yaml

```diff
@@ -0,0 +1,24 @@
+log_format_info: "[%(asctime)s] ${environment} custom log format - %(message)s"
```

Comment:
In `stacker.logger.setup_logging` we also support `color` and `debug` keys. Should we just make this a nested structure.

**Example**

```yaml
log_format:
  info: "[%(asctime)s] ${environment} custom log format - %(message)s"
  color: # Some color format
  debug: # Some debug format
```